_includes/charts/tigera-operator/templates/tigera-operator/02-tigera-operator.yaml

```diff
@@ -42,3 +42,43 @@ spec:
               value: "tigera-operator"
             - name: TIGERA_OPERATOR_INIT_IMAGE_VERSION
               value: {{.Values.tigeraOperator.version}}
+{{- if eq .Values.kubernetesProvider "openshift" }}
+      initContainers:
+        - name: create-initial-resources
+          image: {{ join ":" (list .Values.calicoctl.image .Values.calicoctl.tag) }}
+          env:
+            - name: DATASTORE_TYPE
+              value: kubernetes
+          command:
+            - /bin/sh
+            - -c
+            - |-
+              if [ ! -d /calicoctl-resources ]; then
```

Comment:
It would be nice if we could put this script into a configmap - I think that would be easier to maintain than the inline script here. WDYT?